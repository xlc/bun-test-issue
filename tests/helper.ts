import { test, expect } from 'bun:test'

export const buildTest = (name: string) => {
    test(name, () => {
        expect(name).toMatchSnapshot()
    })
}