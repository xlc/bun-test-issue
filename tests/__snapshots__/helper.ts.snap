// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`b 1`] = `"b"`;
